README.rst

```diff
@@ -73,3 +86,5 @@ The `vcenter-operator` can be tested as follows:
 - Setup your environment to have access to the desired k8s cluster to test on
 - Run the operator in dry run mode `vcenter-operator --dry-run`
 - This will log the rendered templates and also test the apply functionality in dry-run mode
+
+For the service-user management exist unit-tests that can be run with `pytest tests`
```

Comment:
We should add that removing a CR for a ServiceUser does _not_ delete it from Vault or the vCenters. If we ever need cleanup, we have to do that manually.